filer/migrations/0001_initial.py

```diff
@@ -148,7 +145,7 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='Image',
             fields=[
-                ('file_ptr', models.OneToOneField(serialize=False, auto_created=True, to='filer.File', primary_key=True, parent_link=True, on_delete=django.db.models.deletion.CASCADE)),
+                ('file_ptr', models.OneToOneField(serialize=False, auto_created=False, to='filer.File', primary_key=True, parent_link=True, on_delete=django.db.models.deletion.CASCADE)),
```

Comment:
@yakky is that change required in all other addons as well? Like when it's a "OneToOneField" change auto_created to False?

Reply:
Sorry, the change from `auto_created=True` to `auto_created=False` is not needed (it's been left by error in my commits)
, only `parent_link=True` where missing is required